src/dotnet/App.Maui/Platforms/iOS/IOSNotificationPermissions.cs

```diff
@@ -0,0 +1,15 @@
+using ActualChat.Notification.UI.Blazor;
+
+namespace ActualChat.App.Maui;
+
+public class IOSNotificationPermissions : INotificationPermissions
```

Comment:
So... Let's spell `IOS` as `Ios`, i.e. view it as 3-letter acronym, coz technically it's a misfit for any rule they have: it's not an acronym + not a regular word.